<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteSharedMemoryClass</class>
 <widget class="QMainWindow" name="WriteSharedMemoryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WriteSharedMemory</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>52</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始循环写</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>init</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detachBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>detach</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WriteSharedMemory.qrc"/>
 </resources>
 <connections/>
</ui>
